src/BookNavigator/sharing.js

```diff
@@ -1,6 +1,6 @@
 import { html } from 'lit';
-import '@internetarchive/icon-share/icon-share';
-import '@internetarchive/ia-sharing-options';
+import { iauxShareIcon } from '@internetarchive/ia-item-navigator';
```

Comment:
This is a bit strange for a top-level export ; can we do something like this?

```suggestion
import { iauxShareIcon } from '@internetarchive/ia-item-navigator/sharePanel';
```